R/to_md.R

```diff
@@ -26,56 +26,68 @@
 #' xml2::xml_set_attr(headers3, "level", 1)
 #' yaml_xml_list$body <- body
 #' # save back and have a look
-#' to_md(yaml_xml_list, "newmd.md")
+#' newmd <- tempfile("newmd", fileext = ".md")
+#' to_md(yaml_xml_list, newmd)
 #' # file.edit("newmd.md")
+#' file.remove(newmd)
```

Comment:
much better! should I submit this to CRAN one day? If I do you'll have to become an author.

Reply:
It would be nice to have on CRAN at some point, but I might like to give #9 a shot first :wink: